test/system/CMakeLists.txt

```diff
@@ -14,4 +14,4 @@ target_link_libraries(irohad_test
         )
 add_dependencies(irohad_test irohad)
 add_definitions(-DPATHIROHAD="${PROJECT_BINARY_DIR}/bin")
-add_definitions(-DPATHEXAMPLE="${PROJECT_SOURCE_DIR}/example")
+add_definitions(-DPATHTESTDATA="${CMAKE_CURRENT_SOURCE_DIR}/irohad_test_data")
```

Comment:
Should these files be moved? I guess they probably still remain in example folder

Reply:
I left there the files that I think are suitable as an "example", and removed the ones that were only needed for testing.